apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -2132,3 +2098,54 @@ func deleteCompletedJobs(namespace string) error {
 	}
 	return nil
 }
+
+func updateConfMapWithOwner(r *ReconcileAPI, owner []metav1.OwnerReference, configMap *corev1.ConfigMap) error {
```

Comment:
We'll add method comments to the newly added functions in this PR to improve the readability of the code.

Reply:
sure